freightliner_cascadia_2012_dot_10003/docker-compose.yml

```diff
@@ -156,6 +156,17 @@ services:
     image: usdotfhwastoldev/carma-velodyne-lidar-driver:develop
     container_name: velodyne-lidar-driver
     network_mode: host
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+    command: bash -c 'source /opt/carma/install/setup.bash && ros2 launch /opt/carma/vehicle/config/drivers.launch.py drivers:=velodyne_lidar_driver_wrapper'
```

Comment:
Should this be `... source /opt/carma/install_ros2/setup.bash ..`?

Reply:
No this image contains only a ROS2 installation so it doesn't have the separate install vs install_ros2 folders

Reply:
Okay that makes sense. The rest of the PR looks good so I will approve.